explorer/src/cli/clnodes.ts

```diff
@@ -0,0 +1,103 @@
+import fetch from 'node-fetch'
+import httpStatus from 'http-status-codes'
+import {
+  ADMIN_USERNAME_HEADER,
+  ADMIN_PASSWORD_HEADER,
+} from '../utils/constants'
+
+const EXPLORER_BASE_URL =
+  process.env.EXPLORER_BASE_URL || 'https://explorer.chain.link'
+const EXPLORER_ADMIN_USERNAME = process.env.EXPLORER_ADMIN_USERNAME
+const EXPLORER_ADMIN_PASSWORD = process.env.EXPLORER_ADMIN_PASSWORD
+
+interface CreateChainlinkNode {
+  name: string
+  url?: string
+}
+
+interface CreateChainlinkNodeOk {
+  id: string
+  accessKey: string
+  secret: string
+}
+
+function logError(msg: string) {
```

Comment:
This seems inconsistent with us using `console.error` in other places rather than wrapping it. Maybe we should remove this for now?

Reply:
It was addressing one of the comments. It's for shared error messages.